stix2/test/test_versioning.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import stix2
+
+
+EXPECTED = """{
+    "created": "2016-04-06T20:03:00.000Z",
+    "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+    "description": "Campaign by Green Group against a series of targets in the financial services sector.",
+    "id": "campaign--8e2e2d2b-17d4-4cbf-938f-98ee46b3cd3f",
+    "modified": "2016-04-06T20:03:00.000Z",
+    "name": "Green Group Attacks Against Finance",
+    "type": "campaign"
+}"""
+
+
+def test_making_new_version():
+    campaign_v1 = stix2.Campaign(
```

Comment:
The fixtures can be used any time we want to have a "fixed" value that we don't want to create as part of the test itself. It's partially an code-organization thing (don't repeat yourself), and partially because, if we ever need an "example" Campaign for another test anywhere else, we can use it.